p4-16/psa/PSA.mdk

```diff
@@ -457,7 +457,7 @@ enum HashAlgorithm {
   crc32_custom,
   crc16,
   crc16_custom,
-  random,           /// are random hash algorithms useful?
+  target_default,    /// target implementation defined
```

Comment:
target_default is fine.

I would move identity to be always the first. 

The meaning of *_custom is not quite clear...